src/app/users/users-routing.module.ts

```diff
@@ -42,14 +43,23 @@ const routes: Routes = [
         },
         {
           path: ':id',
-          data: { title: extract('View User'), routeResolveBreadcrumb: ['user', 'username'] },
-          resolve: {
-            user: UserResolver
-          },
           children: [
             {
               path: '',
-              component: ViewUserComponent
+              data: { title: extract('View User'), routeResolveBreadcrumb: ['user', 'username'] },
+              component: ViewUserComponent,
+              resolve: {
+                user: UserResolver
+              }
+            },
+            {
```

Comment:
![Screenshot from 2020-05-23 19-55-35](https://user-images.githubusercontent.com/54709463/82733131-729ae900-9d2f-11ea-8083-847b8d79ccfc.png)
Except for `view/:id` keep it `:id` , rest everything should be same.

Reply:
@karantakalkar If I move the **resolve** outside the children, the view of the user does not get updated after a successful edit. I updated the rest.

Reply:
ok